libs/perun/openapi/src/lib/model/serviceState.ts

```diff
@@ -14,11 +14,12 @@ import { Service } from './service';
 import { Facility } from './facility';
 
 
-export interface ServiceState { 
+export interface ServiceState {
```

Comment:
Dont understand the changes in whole class. Is this generated by openapi?

Reply:
![image](https://user-images.githubusercontent.com/38562381/86337899-fd85d200-bc51-11ea-93b0-7c94a35925ea.png)
ServiceState was genereted with wrong properties. This was meant to be quick fix, but since it worked I forgot about it
(picture is from browser network log)